File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.py
def solution(numbers):
    answer = []
    num = 0
    if(len(numbers) >2):
        for i in range(0, len(numbers)):
            for j in range(i+1,len(numbers)):
                answer.append(numbers[i]+numbers[j])
    else:
        answer.append(numbers[0] + numbers[1])
        
    answer = list(set(answer))
    answer.sort()
    return answer

File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.java
import java.util.*;
class Solution {
    public int[] solution(int[] numbers) {
        
        List<Integer> arr = new ArrayList<Integer>();
        
        for(int i = 0; i < numbers.length; i++){
            for(int j = i+1; j < numbers.length; j++){
                if(!(arr.contains(numbers[i] + numbers[j]))){
                    arr.add(numbers[i] + numbers[j]);
                }
            }
        }
        int [] answer = arr.stream().mapToInt(i -> i).toArray();
        Arrays.sort(answer);
        
        
        return answer;
    }
}